data/campaigns/The_Rise_Of_Wesnoth/scenarios/17a_The_Dragon.cfg

```diff
@@ -25,21 +25,24 @@
         unrenamable=yes
         side=1
         canrecruit=yes
-        {GOLD 230 200 170}
+        {GOLD 290 230 170}
+        {INCOME 6 4 2}
```

Comment:
I'd say drop the income here. This is supposed to be the hardest scenario in the quest batch. If you played this scenario, I'd like to see the replay with your changes in (or if you play my original rebalance that's fine too).

Reply:
ok 
[TRoW-The Dragon replay.gz](https://github.com/wesnoth/wesnoth/files/6133748/TRoW-The.Dragon.replay.gz)